### ISSUE TITLE
Split JWD into separate library

### ISSUE THREAD

**Issue by EternalDeiwos** [anvilresearch/jwt#7]
With npm allowing us to use namespaced package names and our current theme of _@trust/jwX_ packages, it may be desirable for us to split JWD into it's own package that depends on _@trust/jwt_.

If we do decide to do this then we should probably only do it once JWT is refactored and working as intended.

Thoughts?

**Comment by christiansmith** [comment]
I strongly agree about JWD eventually becoming it's own package. Not sure about the order of extraction or initially grouping JWD here. Thinking it will become apparent as we sort out some of the current design challenges.
